ligo/stablecoin/actions.ligo

```diff
@@ -98,7 +98,10 @@ function transfer
 } with (upd, ups)
 
 (*
- * Returns a list of `balance_of` calls and leaves the storage untouched
+ * Retreives a list of `balance_of` calls that is specified for multiple
```

Comment:
The right spelling is "Retrieves", it should be fixed in many places. And I still don't understand why you say "a list of calls". I assume by "call" you mean `TRANSFER_TOKENS` operation and this function returns just one call. I mean, technically it's still a list (with one item), but imo this "list" is confusing.